src/ladds/Simulation.cpp

```diff
@@ -227,7 +272,8 @@ size_t Simulation::simulationLoop(AutoPas_t &autopas,
   const auto [hdf5WriteFrequency, hdf5Writer, conjuctionWriter] = initWriter(config);
 
   // set constellation particle IDs and fetch maxExistingParticleId
-  const size_t maxExistingParticleId = setConstellationIDs(autopas, constellations);
+  setConstellationIDs(autopas, constellations);
+  const size_t maxExistingParticleId = autopas.getNumberOfParticles();
```

Comment:
with the new naming, is the max actually still the number of particles?

Reply:
here yes but since the variable exists way beyond this and might be abused at some point we should directly use the result of the function. -> a7d9672